tests/int/tests/medic-webapp.js

```diff
@@ -28,6 +28,32 @@ describe('Basic Medic-Webapp smoke test (v. slow tests!)', function() {
     });
   });
 
+  it('should --install an upgrade without error', () => {
+    return hortiUtils.start([
+      '--install=medic:medic:3.0.x',
+      '--test'
+    ], {
+      waitUntil: /Watching for deployments/,
+      buildServer: PROD_BUILD_URL,
+      log: true
+    }).then(horti => {
+      horti.kill();
+    });
+  });
+
+  it('should --install a second upgrade without error', () => {
```

Comment:
Do `it` statements always run in the order that they're stated? I know in some languages / frameworks that's not an explicit relationship. Do you think it would make more sense for the entire test (both installs) to be inside one `it`?

Reply:
I merged the two installs inside one `it`. The sequence really didn't matter in this case, I only wanted to trigger the symlinks overwrite - the only requirement was to install 3 different versions (in this case, `master`, `3.0` and `3.1`).  

Also, it seems the integration tests already relied on them being executed in sequence (if the `should start the daemon with no install without error` ran before `should --install master without error`, it would fail as there is no API to start).